projects/angular-tags-input/src/lib/dropdown/dropdown.component.html

```diff
@@ -1,31 +1,43 @@
-
-<ng-container
- *ngTemplateOutlet="dropDownTemplate; context: context">
+<ng-container *ngIf="!noRecordsFound; else noRecordsTemplate">
+  <ng-container
+    *ngTemplateOutlet="dropDownTemplate; context: context"
+  ></ng-container>
```

Comment:
**suggestion:** We can revert to how this was before. Please see my other comments